Surface_mesh_simplification/doc/Surface_mesh_simplification/Concepts/GetFilter.h

```diff
@@ -0,0 +1,40 @@
+/*!
+\ingroup PkgSurfaceMeshSimplificationConcepts
+\cgalConcept
+
+The concept `GetFilter` describes the requirements for the <I>policy
+function object</I> which gets the profile and placement of an edge
+and which can filter the placement.  The filtering is only done when
+an edge is taken from the priority queue in order to get collapsed,
+and neither when the edge is inserted or updated in the priority queue.
+
+The placement returned is a `boost::optional` value (i.e., it can
+be absent). The value `boost::none` indicates that the edge should not be collapsed.
+
+\cgalRefines `DefaultConstructible`
+\cgalRefines `CopyConstructible`
+
+\cgalHasModel `CGAL::Surface_mesh_simplification::Bounded_normal_change_filter<BaseFilter>`
+\cgalHasModel `CGAL::Surface_mesh_simplification::Envelope_filter<GeomTraits, BaseFilter>`
+*/
+
+
+class GetFilter
```

Comment:
Where does the name `GetFilter` comes from ?

Reply:
The `Get` comes from `GetCost`, but  to take this prefix makes maybe no sense.